cycles/cycles/Program.cs

```diff
@@ -0,0 +1,348 @@
+            {
+
+            }
+            foreach (var item in input)
+            {
+                result++;
+            }
+            Console.WriteLine($"Длинна строки: {input.Length}");
+
+            Console.WriteLine(result);
+        } // Write a program in C# Sharp to find the length of a string without using library function
+        static void SpaceText() // Write a program in C# Sharp to separate the individual characters from a string
+        {
+            Console.WriteLine("Введите что то:");
+            string input = Console.ReadLine();
+            int result = 0;
+
+            while (result <= input.Length - 1)
+            {
+                Console.Write(input[result] + " ");
+                result++;
+            }
+        }
+        
+        static void CountWords()
+        {
+            Console.WriteLine("Введите что то:");
+            string input = Console.ReadLine();
+            int result = 1;
+            char space = ' ';
+            for (int i = 0; i < input.Length; i++)
+            {
+                if (input[i] == space)
+                {
+                    result++;
+                }
+
+            }
+            Console.WriteLine($"Количество слов в тексте: {result}");
+        } // Write a program in C# Sharp to count the total number of words in a string
+
+        static void ComapreStrings()
+        {
+            int resultWords = 0;
+            Console.WriteLine("Введите что то 1:");
+            var input = Console.ReadLine();
+            Console.WriteLine("Введите что то 2:");
+            var input2 = Console.ReadLine();
+            for (int a = 0; a < input.Length; a++) // сравниваем буквы в двух массивах
+            {
+                for (int b = a; b < input2.Length; b++)
+                {
+                    if (input[a] == input2[b])
+                    {
+
+                        resultWords++;
+                        break;
+                    }
+                }
+            }
+            if (resultWords == input.Length & resultWords == input2.Length) // сравниваем длинну c количеством правильных буквы
+            {
+                Console.WriteLine("Ура!!!! СТРОКИ СОВПАДАЮТ!");
+            }
+            else
+            {
+                Console.WriteLine("Не ура!!!! СТРОКИ НЕ СОВПАДАЮТ)");
+            }
+        } // Write a program in C# Sharp to compare two string without using string library functions
+
+        static void CoppyString() // Write a program in C# Sharp to copy one string to another string
+        {
+            Console.WriteLine("Введите что то:");
+            var input = Console.ReadLine();
+            string result = "";
+            char word;
+            int i = 0;
+            for (i = 0; i < input.Length; i++)
+            {
+
+                word = input[i];
+                result = result + word;
+
+
+            }
+            Console.WriteLine($"{result} был скопирован. Количесво букв: {i}");
```

Comment:
форматирование прям совсем беда
```suggestion
            {
                word = input[i];
                result = result + word;
            }
            
            Console.WriteLine($"{result} был скопирован. Количесво букв: {i}");
```